torch/_ops.py

```diff
@@ -127,6 +127,110 @@ def inner(fn):
 
         return inner
 
+    # Registers an implementation to all **3** variants of functionalization that we have:
+    # - DispatchKey.Functionalize
+    # - functorch.TransformType.FuncTorchFunctionalize
+    # - FunctionalTensorMode
+    def py_functionalize_impl(self):
```

Comment:
Could you add an example of how to use it in the comment?

Reply:
will do